modules/parquet/src/parquetjs/compression.ts

```diff
@@ -46,9 +46,9 @@ const modules = {
   //     throw new Error('brotli compress');
   //   }
   // },
-  lz4js
+  lz4js,
   // lzo
-  // 'zstd-codec': ZstdCodec
+  'zstd-codec': ZstdCodec
```

Comment:
- We can't enable this codec by default, it is not only enormous (2.5MB), it breaks the website bundling. 
- We need to find a way to let the application provide it. There is an undocumented function `preloadCompressions({modules: {['zstd-codec': ZstdCodec)` that can be used by the application.
- Also a pre-built worker loader could have this bundled in.

Reply:
Done